src/helperFunctions/web_interface.py

```diff
@@ -60,9 +60,10 @@ def get_template_as_string(view_name):
 
 
 def get_radare_endpoint(config):
+    radare2_host = config['ExpertSettings']['radare2_host']
     if config.getboolean('ExpertSettings', 'nginx'):
-        return 'https://localhost/radare'
-    return 'http://localhost:8000'
+        return 'https://' + radare2_host + '/radare'
+    return 'http://' + radare2_host + ':8000'
```

Comment:
Since we agreed internally to use `.format()` for string formatting, it would be great if you could change the lines in that way to keep the style consistent.